app/Http/Controllers/Admin/BookController.php

```diff
@@ -3,13 +3,11 @@
 namespace App\Http\Controllers\Admin;
 
 use Illuminate\Http\Request;
-use App\Http\Requests\ValidationCategory;
+use App\Http\Requests\ValidationBook;
 use App\Http\Controllers\Controller;
-use Session;
-use App\Models\Category;
-use Illuminate\Support\Facades\Config;
-
-class CategoryController extends Controller
```

Comment:
E ơi. E check lại giúp chị Controller này e nhé. Vì chị thấy ở đây e xóa mất CategoryController rồi thì nếu chị merge vô sẽ mất toàn bộ hần category của Tiên. Chỗ này chắc e bị nhầm lẫn về git rồi. E nên đứng từ branch develop, pull về mới nhất r checkout ra và làm tiếp nhánh của mình hì